api/v1/type.go

```diff
@@ -98,7 +98,7 @@ type Release struct {
 	CreatedAt    time.Time       `json:"_createdAt" binding:"-"`
 	UpdatedAt    time.Time       `json:"_updatedAt" binding:"-"`
 	DeletedAt    *time.Time      `json:"-" binding:"-"`
-	BadgeRatings *postgres.Jsonb `json:"-" binding:"-"`
+	BadgeRatings *postgres.Jsonb `json:"badgeratings" binding:"-"`
```

Comment:
What is the reason behind this change ?
I remember we said that we should not show badge ratings on the app's resource.

Reply:
Done